setting-up-bacalhau-cluster/setting-up-bacalhau-network-aws-spot/spot_creation_scripts/100_install_bacalhau.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+BACALHAU_INSTALL_ID=${BACALHAU_INSTALL_ID:-"BACA14A0-eeee-eeee-eeee-eeeeeeeeeeee"}
```

Comment:
As mentioned, none of these scripts should be used. We should only use the Docker in Docker version of bacalhau.